dimod/sampleset.py

```diff
@@ -453,6 +453,27 @@ def infer_vartype(samples_like):
 
     raise ValueError("given samples_like is of an unknown vartype")
 
+def drop_variables(sampleset, variables_to_drop):
+    """Drop variables from a sampleset
+
+    Args:
+        sampleset (:obj:`.SampleSet`):
```

Comment:
One should [use `class`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects) for a specific type.
```suggestion
        sampleset (:class:`.SampleSet`):
```